PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalAccount.kt

```diff
@@ -0,0 +1,65 @@
+package com.braintreepayments.api.paypal
+
+import com.braintreepayments.api.core.PaymentMethod
+import org.json.JSONException
+import org.json.JSONObject
+
+/**
+ * Use to construct a PayPal account tokenization request
+ *
+ * @property clientMetadataId Application clientMetadataId created by
+ * [DataCollector.getClientMetadataId]. Used by PayPal wrappers to construct a request to create a
+ * PayPal account.
+ * @property urlResponseData Response data from callback url. Used by PayPal wrappers to construct a
+ * request to create a PayPal account. Response data will be merged into the payment method json on
+ * [buildJSON].
+ * @property intent Used by PayPal wrappers to construct a request to create a PayPal account.
+ * @property merchantAccountId String merchant account id
+ * @property paymentType Payment type from original PayPal request. Either "billing-agreement" or
+ * "single-payment"
+ */
+internal data class PayPalAccount(
+    val clientMetadataId: String?,
+    val urlResponseData: JSONObject,
```

Comment:
General style question, do we want to consider having required properties at the top followed by optional?

Reply:
Yep! I think having all required fields at the top makes sense. In this case, all of these fields are required as there are no default values defined. Although most of them are marked as nullable.

Reply:
This is actually an internal class, so we'd be the ones to invoke the constructor 😄 Going to merge the PR as is for now but happy to keep the convo for this open!